web/src/HUD.tsx

```diff
@@ -464,6 +466,12 @@ class HUD extends Component<HudProps, HudState> {
     let tiltCloudUsername = (view && view.TiltCloudUsername) || null
     let tiltCloudSchemeHost = (view && view.TiltCloudSchemeHost) || ""
     let tiltCloudTeamID = (view && view.TiltCloudTeamID) || null
+    let highlightedLines = this.state.snapshotHighlight
+      ? Math.abs(
+          parseInt(this.state.snapshotHighlight.endingLogID) -
```

Comment:
```suggestion
          parseInt(this.state.snapshotHighlight.endingLogID, 10) -
```

I'm pretty sure this needs to specify the radix, because the [radix is not guaranteed to be 10 if it isn't specified](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/parseInt).